doc/tutorials/ann/ann.txt

```diff
@@ -253,7 +253,9 @@ void GenerateNoisySines(arma::mat& data,
         arma::as_scalar(arma::randu(1) - 0.5) * noise;
     labels(1, sequences + seq) = 1;
   }
-
+}
```

Comment:
Can you put a blank line between the end of `GenerateNoisySines()` and `RNNModel()`?

Reply:
Sure